roles/pcp/README.md

```diff
@@ -88,9 +88,9 @@ Basic PCP setup with monitoring suited for a single host.
   roles:
     - role: performancecopilot.metrics.pcp
       vars:
-        pcp_pmlogger_interval: 10
-        pcp_optional_agents: [dm, nfsclient, openmetrics]
-        pcp_explicit_labels:
+    ### pcp_pmlogger_interval: 10
+    ### pcp_optional_agents: [dm, nfsclient, openmetrics]
+    ### pcp_explicit_labels:
           environment: production
```

Comment:
not sure what markdownlint was complaining about here, but these should not be commented out

Reply:
@spetrosi I think the change is fine for all except those last three lines @richm has called out - this is an example and I think these have accidentally been turned into comments here?

Reply:
You're right, it was some incorrect mass replace on my side. Now it's fixed.